### PR TITLE
019 form 2.0

### PR DESCRIPTION
**Pull request by Hamza-Luciano**

### CODE REVIEW

**Review by FrancoisSalmon — APPROVED** [review]
Clean old import

File: frontend/src/components/Form/index.jsx [hunk 1]
```diff
@@ -10,11 +17,35 @@ export default function Form() {
         ci-dessous. Nous vous apporterons une réponse rapidement.
       </p>
       <form action="">
```
Comment (LuigiMenez) [inline frontend/src/components/Form/index.jsx:19]: Why is the action empty?

**Review by LuigiMenez — APPROVED** [review]
Ok

File: frontend/src/components/Form/index.jsx [hunk 2]
```diff
@@ -1,7 +1,14 @@
+import React, { useState } from "react";
 import Title from "@components/Archi/Title";
+import Modal from "@components/Modal";
 import SForm from "./style";
 
 export default function Form() {
+  const [message, setMessage] = useState("");
+  const [name, setName] = useState("");
+  const [object, setObject] = useState("");
+  const [email, setEmail] = useState("");
```
Comment (LoicBrassart) [inline frontend/src/components/Form/index.jsx:10]: We saw how to create one state to store all field's information, to make the next steps easier

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
- Answer Luigi's question
- Your commit messages don't tell anything useful
- You must code the backend along with your frontend

**Pull request closed without merging**